### PR TITLE
fix(yield-temp): Explain how to calculate APY

### PR DESCRIPTION
**Pull request by nicholaspai**
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

### CODE REVIEW

**Review by daywiss — APPROVED** [review]

File: features/yield/Yield.tsx [hunk 1]
```diff
@@ -124,6 +124,18 @@ const Yield = () => {
...
+            <br></br>
+            <br></br>
+            <Typography>
+              <strong>APY</strong>: To approximate your weekly returns, you can
+              divide the number of UMA rewarded for a pool per week by the $
+              amount of pooled liquidity. For example, if the pool size is $10
+              million and the reward is 25,000 UMA/week, then the return for
+              each $1 supplied to the pool is:{" "}
+              <strong>
+                (25,000 / $10 million) = 0.0025 UMA per week per $1
```
Comment (mrice32) [inline features/yield/Yield.tsx:136]: Do you want to also add the UMA price in there to make it a typical APY percentage? Something like:
```
($10 * 25,000 / $10 million) * 100 = 2.5% per week
```

**Review by mrice32 — APPROVED** [review]
Looks great! One minor suggestion

File: features/yield/Yield.tsx (hunk 1 above)
Comment (nicholaspai) [reply features/yield/Yield.tsx:136]: great suggestion

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**